src/app/js/formats/streamgraph/utils.js

```diff
@@ -177,18 +177,25 @@ export function getMinMaxValue(stackedData) {
     return { minValue, maxValue };
 }
 
-export function cutStr(str) {
-    const strSplit = str.split(' ');
+export function cutStr(str, column_number) {
+    const strSplit = str.split(/-| /);
```

Comment:
You also could use a character class, instead of a `|`.

```suggestion
    const strSplit = str.split(/[- ]/);
```

But this is only a matter of taste.